schedule/week_2/xquery_1.md

```diff
@@ -172,11 +172,13 @@ let $count := count($distinct)
 return $count
 ```
 
-No one would ever write such a simple expression this way, but this example shows you how you can create individual steps by declaring variables. During development, we might write each step separately and return the result to verify that we are asking for what we think we’re asking for. If we wanted to change the code to focus on a single scene, instead of the entire play, or to get us the distinct speakers for each act, or to count the speakers by act, we could do so by adding a few lines and editing the result.
```

Comment:
"No one would ever..." is a little heavy-handed in phrasing, particularly for beginners. Anyway, to take the reverse position, these "baby steps" are exactly what we expect new coders to write.